go.mod

```diff
@@ -58,6 +59,8 @@ require (
 	github.com/VividCortex/ewma v1.2.0 // indirect
 	github.com/a8m/envsubst v1.4.2 // indirect
 	github.com/alecthomas/participle/v2 v2.1.1 // indirect
+	github.com/bmatcuk/doublestar/v4 v4.6.0 // indirect
+	github.com/braydonk/yaml v0.7.0 // indirect
```

Comment:
Not a huge fan of importing multiple YAML libraries

Reply:
Is this the fifth YAML library?

Reply:
Oh, is this the first time a YAML library forked from https://github.com/go-yaml/yaml is being added?

Reply:
k8s yaml is a wrapper around go-yaml, as far as I know... <https://github.com/kubernetes-sigs/yaml>

Reply:
I want to express my gratitude to Go for allowing the use of both a forked library and the original library simultaneously without causing issues. 👏🏻